src/components/NavigationMenu.tsx

```diff
@@ -1,31 +1,40 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 
 import '../styles/components/NavigationMenu.css';
 
 import SiteLogo from '../components/SiteLogo';
 
 export default function NavigationMenu() {
     const [useLogo, setUseLogo] = useState(false);
+    const [currentPage, setCurrentPage] = useState('page-landing');
 
+    const isActive = (element_id:string) =>{
+        if(element_id == currentPage){
```

Comment:
Lint? :D

Reply:
It's formatted  @lucasmauro, accordingly with the most used linter for react (eslint) 😢 
Ps.: I've added a lint check on Travis CI as part of the CI workflow, it checks the lint and then run the tests).